ui/component/or-mwc-components/src/or-mwc-table.ts

```diff
@@ -1,14 +1,14 @@
-import {css, html, LitElement, TemplateResult, unsafeCSS, PropertyValues} from "lit";
+import {css, html, LitElement, TemplateResult, unsafeCSS} from "lit";
 import {customElement, property, state} from "lit/decorators.js";
 import {classMap} from "lit/directives/class-map.js";
 import {until} from 'lit/directives/until.js';
 import {MDCDataTable} from "@material/data-table";
-import {MDCCheckbox} from "@material/checkbox";
 import {when} from 'lit/directives/when.js';
-import {DefaultColor3, DefaultColor2, DefaultColor1, manager} from "@openremote/core";
+import {DefaultColor3, DefaultColor2, DefaultColor1} from "@openremote/core";
 import {i18next} from "@openremote/or-translate";
 import {InputType, OrInputChangedEvent} from "./or-mwc-input";
 import { styleMap } from "lit/directives/style-map.js";
+import moment from "moment";
```

Comment:
Not a fan importing `moment` here as it's a big library, and should not be a dependency of the table on itself.
Feel free to use a (temporarily) different strategy for adjusting / formatting dates in `or-mwc-table`.